examples/40_advanced/example_pandas_train_test.py

```diff
@@ -118,26 +118,30 @@ def get_runhistory_models_performance(automl):
 # ===================================
 
 ensemble_performance_frame = pd.DataFrame(cls.automl_.ensemble_performance_history)
-best_values = pd.Series({'ensemble_optimization_score': -np.inf,
-                         'ensemble_test_score': -np.inf})
+
+best_values = pd.Series({
+    'ensemble_optimization_score': -np.inf,
+    'ensemble_test_score': -np.inf
+})
 for idx in ensemble_performance_frame.index:
-    if (
-        ensemble_performance_frame.loc[idx, 'ensemble_optimization_score']
-        > best_values['ensemble_optimization_score']
-    ):
+    ensemble_score = ensemble_performance_frame.loc[idx, 'ensemble_optimization_score']
```

Comment:
I disagree on this change - isn't the former one easier to read and also preferable as it doesn't create an intermediate variable?

Reply:
I just found it very crowded personally but I'm okay with either. I'll change it back